DOC: faulty example on Solving Package Environments page

### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### Anaconda default channels

- [X] I do NOT use the Anaconda default channels (pkgs/* etc.)

### How did you install Mamba?

Micromamba

### Search tried in issue tracker

yes

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

I didn't try

### Describe your issue

Example taken from https://mamba.readthedocs.io/en/latest/usage/solver.html#populating-the-package-database:

```python
import libmambapy
db = libmambapy.solver.libsolv.Database(
    libmambapy.specs.ChannelResolveParams(channel_alias="https://conda.anaconda.org")
)
```

Output:

```
E       TypeError: __init__(): incompatible constructor arguments. The following argument types are supported:
E           1. libmambapy.bindings.specs.ChannelResolveParams(platforms: set[str] = set(), channel_alias: libmambapy.bindings.specs.CondaURL = https://localhost/, custom_channels: libmambapy.bindings.specs.ChannelResolveParams.ChannelMap = <libmambapy.bindings.specs.ChannelResolveParams.ChannelMap object at 0x105fb53f0>, custom_multichannels: libmambapy.bindings.specs.ChannelResolveParams.MultiChannelMap = <libmambapy.bindings.specs.ChannelResolveParams.MultiChannelMap object at 0x105fd8470>, authentication_db: libmambapy.bindings.specs.AuthenticationDataBase = <libmambapy.bindings.specs.AuthenticationDataBase object at 0x105fd84b0>, home_dir: str = '', current_working_dir: str = '')
E       
E       Invoked with: kwargs: channel_alias='https://conda.anaconda.org'
```

### mamba info / micromamba info

```shell
libmamba version : 2.0.2
     micromamba version : 2.0.2
           curl version : libcurl/8.10.1 (OpenSSL/3.3.2) SecureTransport zlib/1.3.1 zstd/1.5.6 libssh2/1.11.0 nghttp2/1.58.0
     libarchive version : libarchive 3.7.4 zlib/1.2.13 bz2lib/1.0.8 libzstd/1.5.6
       envs directories : /Users/lucascolley/micromamba/envs
          package cache : /Users/lucascolley/micromamba/pkgs
                          /Users/lucascolley/.mamba/pkgs
            environment : asv-dev (active)
           env location : /Users/lucascolley/micromamba/envs/asv-dev
      user config files : /Users/lucascolley/.mambarc
 populated config files : /Users/lucascolley/.condarc
       virtual packages : __unix=0=0
                          __osx=14.6.1=0
                          __archspec=1=arm64
               channels : https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/conda-forge/osx-arm64
                          https://conda.anaconda.org/nodefaults/noarch
                          https://conda.anaconda.org/nodefaults/osx-arm64
       base environment : /Users/lucascolley/micromamba
               platform : osx-arm64
```


### Logs

_No response_

### environment.yml

_No response_

### ~/.condarc

_No response_

--- a/docs/source/usage/solver.rst
+++ b/docs/source/usage/solver.rst
@@ -41,8 +41,10 @@ The first way to add a repository is from a list of |PackageInfo| using
 
    import libmambapy
 
+   channel_alias = libmambapy.specs.CondaURL.parse("https://conda.anaconda.org")
+
    db = libmambapy.solver.libsolv.Database(
-       libmambapy.specs.ChannelResolveParams(channel_alias="https://conda.anaconda.org")
+       libmambapy.specs.ChannelResolveParams(channel_alias=channel_alias)
    )
 
    repo1 = db.add_repo_from_packages(

--- a/docs/source/usage/specs.rst
+++ b/docs/source/usage/specs.rst
@@ -151,11 +151,12 @@ All parameters that influence this resolution must be provided explicitly.
 .. code:: python
 
    import libmambapy.specs as specs
+   import libmambapy.specs.CondaURL as CondaURL
 
    uc = specs.UnresolvedChannel.parse("conda-forge[prius-avx42]")
    chan, *_ = specs.Channel.resolve(
        uc,
-       channel_alias="https://repo.mamba.pm"
+       channel_alias=CondaURL.parse("https://repo.mamba.pm")
        # ...
    )
 
@@ -168,11 +169,12 @@ There are no hard-coded names:
 .. code:: python
 
    import libmambapy.specs as specs
+   import libmambapy.specs.CondaURL as CondaURL
 
    uc = specs.UnresolvedChannel.parse("defaults")
    chan, *_ = specs.Channel.resolve(
        uc,
-       channel_alias="https://repo.mamba.pm"
+       channel_alias=CondaURL.parse("https://repo.mamba.pm")
        # ...
    )
 

